.github/workflows/ci.yaml

```diff
@@ -35,4 +35,41 @@ jobs:
         if: always()
         uses: guyarb/golang-test-annotations@v0.5.1
         with:
-          test-results: test.json
\ No newline at end of file
+          test-results: test.json
+  test:
+    runs-on: ubuntu-latest
+    env:
+      BATON_LOG_LEVEL: debug
+      # Add any environment variables needed to run baton-bitbucket-datacenter
+      BATON_DEPLOYMENT_API_KEY: ${{ secrets.BATON_DEPLOYMENT_API_KEY }}
+      BATON_DEPLOYMENT_ENDPOINT: ${{ secrets.BATON_DEPLOYMENT_ENDPOINT }}
+      BATON_ORGANIZATION_ID: ${{ secrets.BATON_ORGANIZATION_ID }}
+      BATON_API_KEY: ${{ secrets.BATON_API_KEY }}
+      # The following parameters are passed to grant/revoke commands
+      CONNECTOR_GRANT: 'roleMapping:mapping7:member:deploymentUser:jacknich'
+      CONNECTOR_ENTITLEMENT: 'roleMapping:mapping7:member'
+      CONNECTOR_PRINCIPAL_TYPE: 'deploymentUser'
+      CONNECTOR_PRINCIPAL: 'jacknich'
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.22.x
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Install baton
+        run: ./scripts/get-baton.sh && mv baton /usr/local/bin
+      - name: Build baton-elastic
+        run: go build ./cmd/baton-elastic
+      - name: Run baton-elastic
+        run: |
+          ./baton-elastic
+          baton grants -t roleMapping
```

Comment:
Was this for debugging? I guess it doesn't hurt, but it's not checking anything except whether the c1z exists.

Reply:
Fixed